app/models/orchestration_template/orchestration_parameter_fqname.rb

```diff
@@ -0,0 +1,5 @@
+class OrchestrationTemplate
+  class OrchestrationParameterFqname < OrchestrationParameterConstraint
```

Comment:
Do you think the `attr_accessor fqname` should also be renamed? I had a bit of a problem with choosing the proper name. `fqname` is used in the `resource_action` of the dialog field.

Patch is updated with the suggested name change.